packages/styles/scss/components/callout-quote/_callout-quote.scss

```
@@ -53,4 +35,70 @@
       outline: transparent;
     }
   }
+
+  :host(#{$c4d-prefix}-callout-quote[color-scheme='inverse']) {
+    @extend :host(#{$c4d-prefix}-callout[color-scheme='inverse']);
+    @extend :host(#{$c4d-prefix}-callout-text[color-scheme='inverse']);
+  }
+
+  :host(#{$c4d-prefix}-callout-quote[color-scheme='layer']) {
+    @extend :host(#{$c4d-prefix}-callout[color-scheme='layer']);
+    @extend :host(#{$c4d-prefix}-callout-text[color-scheme='layer']);
+  }
+
+  :host(#{$c4d-prefix}-callout-quote[color-scheme='purple']) {
+    @extend :host(#{$c4d-prefix}-callout[color-scheme='purple']);
+    @extend :host(#{$c4d-prefix}-callout-text[color-scheme='purple']);
+  }
+  :host(#{$c4d-prefix}-callout-quote[color-scheme='cyan']) {
+    @extend :host(#{$c4d-prefix}-callout[color-scheme='cyan']);
+    @extend :host(#{$c4d-prefix}-callout-text[color-scheme='cyan']);
+  }
+
+  :host(#{$c4d-prefix}-callout-link-with-icon) {
+    @extend .#{$prefix}--link-with-icon;
+
+    display: inline-block;
+    a.#{$prefix}--link-with-icon.#{$prefix}--link.#{$prefix}--link-with-icon__icon-right {
+      color: $link-primary;
+      outline-color: $focus-inverse;
+
+      &:hover {
+        // @carbon/styles has no hover color for $inverse-link.
+        // Solution is to use the same token and add underline
+        color: $link-primary;
```

```suggestion
        color: $link-inverse;
```
Should this be changed?

---

ooh good catch i updated to use `link-primary-hover` actually which was added as a token!